test/sql-tap/sql-errors.test.lua

```diff
@@ -33,7 +33,7 @@ test:do_catchsql_test(
 	create_statement,
 	{
 		-- <sql-errors-1.2>
-		1,"Failed to create space 'T2': space column count 2001 exceeds the limit (2000)"
+        1,"Failed to create space 't2': space column count 2001 exceeds the limit (2000)"
```

Comment:
Typo: I suggest to leave the original indent level.

Reply:
See above.